ios/app/MBXViewController.mm

```diff
@@ -261,6 +264,10 @@ - (void)actionSheet:(UIActionSheet *)actionSheet didDismissWithButtonIndex:(NSIn
     }
     else if (buttonIndex == actionSheet.firstOtherButtonIndex + 9)
     {
+        [self presentAnnotationWithCustomCallout];
+    }
+    else if (buttonIndex == actionSheet.firstOtherButtonIndex + 10)
```

Comment:
This needs a refactor as I've done on another pull request to simplify the indexes!. Anyway, done.